### DISCUSSION TITLE
Issue with installing ASReview on macOS: pre-installed Python version

### DISCUSSION THREAD

**Discussion by elenajal** [asreview/asreview#1650] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
When trying to install ASReview on my MacBook through the terminal I ran into the following issue: MacBooks have an older version of Python pre-installed (in my case ver. 2.7.16). Checking in my terminal for the version of Python (python --version) yielded the older version 2.7.16 and when trying to install ASReview, this older Python version was automatically used as a default for installing instead of the newer version I downloaded (ver. 3.9). How can I set the python version to the correct one so I am able to install ASReview?

**Comment by J535D165** [comment, marked as answer]
Most likely the following works:

```sh 
python3 -m pip install asreview

python3 -m asreview lab
```

You might want to update your MacBook as well, that might resolve the issue for you. 

Another option: https://www.janmeppe.com/blog/how-to-set-python3-as-default-mac/

**Reply by elenajal** [reply]
Thank you for the reply! 

While trying the first option, I ran into another issue. After setting the default python version to 3.9, the terminal returned the following error message: 

> "ERROR: failed building wheel for gevent"

To resolve this issue I used the tool "pyenv" proposed in the second option according to these steps: 

1. install pyenv in terminal: 

`curl https://pyenv.run | bash`

2. set the global version of python to the newer version: 

`pyenv global 3.9`

3. restart terminal

4. Check which version of Python is set as the default: 

`python --version`

5. try to install ASReview again: 

`pip install asreview `

6. Open ASReview: 

`asreview lab`
 
After taking these steps I was able to install and open ASReview on my MacBook!

**Reply by J535D165** [reply]
Is there any reason you used Python version 3.9 and not a more recent version?
